pic-sure-resources/pic-sure-resource-api/pom.xml

```diff
@@ -61,5 +61,11 @@
           <artifactId>swagger-annotations</artifactId>
           <version>2.2.8</version>
       </dependency>
+      <dependency>
```

Comment:
Do we use lombok elsewhere in our codebases? It's a fairly divisive dependency to include.